### PR TITLE
Add filesize to file view

### PR DESCRIPTION
**Pull request by lukechampine**
sister PR of https://github.com/NebulousLabs/Sia/pull/592

### CODE REVIEW

File: site/markup/pages/files.haml [hunk 1]
```diff
@@ -27,6 +27,7 @@
                 .no
                     %i.fa.fa-refresh.fa-spin
             .name
+            .size
```
Comment (DavidVorick) [inline site/markup/pages/files.haml]: this doesn't cause overflow issues?

Comment (DavidVorick) [inline site/markup/pages/files.haml]: just checked, it's fine for me, but I'm a bit surprised.

Comment (lukechampine) [inline site/markup/pages/files.haml]: there was already some markup for `.size` for some reason, `width: 10%` IIRC

Comment (DavidVorick) [inline site/markup/pages/files.haml]: heh

**Pull request merged**